README.md

```diff
@@ -8,305 +8,409 @@
 
-This will use the IBM Cloud ResourceGroup `default`. To specify a different ResourceGroup, use the following command:
-```bash
-ibmcloud target -g <resource-group>
-```
+## Prerequisites
 
-Notice that the `org` and `space` must be included, even if no Cloud Foundry services will be instantiated.
+1.  Have an [IBM Cloud account](https://cloud.ibm.com/registration).
+2.  Have a cluster that runs Kubernetes version 1.11 or later (OpenShift 3.11 or later).
+3.  Install the required command line tools.
+    *   [IBM Cloud CLI](https://cloud.ibm.com/docs/cli?topic=cloud-cli-getting-started) (`ibmcloud`)
+    *   [Kubernetes CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl/) (`kubectl`)
+4.  Log in to your IBM Cloud account from the CLI.
+    
+    ```bash
+    ibmcloud login
+    ```
 
-## Set up the operator
+5.  Target the appropriate environment (`--cf`) and resource group (`-g`). Note that you must still set the Cloud Foundry `org` and `space` environment, even if you do not create any Cloud Foundry services.
+    
+    ```bash
+    ibmcloud target --cf -g <default>
+    ```
 
-To set up the operator after logging in with `ibmcloud`, run the below installer.
+6.  Set the Kubernetes context of your CLI to your cluster so that you can run `kubectl` commands. For example, if your cluster runs OpenShift, use the `oc login` command. 
 
-By default, the script will create a new API key for use in the operator. To use a custom API key, set the `IBMCLOUD_API_KEY` environment variable to the key.
+<!-- END SHOW operator hub -->
 
-#### Using a ServiceId
+If your cluster is in IBM Cloud Kubernetes Service, run the following command.
+    
+    ```bash
+    ibmcloud ks cluster config -c <cluster_name_or_ID>
+    ```
 
-To instantiate services and bindings on behalf of a ServiceId, set the environment variable `IBMCLOUD_API_KEY` to the `api-key` of the ServiceId. This can be obtained via the IBM Cloud Console or [CLI](https://cloud.ibm.com/docs/iam?topic=iam-serviceids). Be sure to give proper access permissions to the ServiceId.
+    To check that your Kubernetes context is set to your cluster, run the following command.
+    
+    ```bash
+    kubectl config current-context
+    ```
 
-Next log into the IBM Cloud account that owns the ServiceId and follow the instructions above.
+[Back to top](#ibm-cloud-operator)
+<!-- SHOW operator hub -->
 
-<!-- END SHOW operator hub -->
+## Setting up the operator
 
-### Install
+You can use an installation script to set up the IBM Cloud Operator. The installation script stores an API key in a Kubernetes secret in your cluster that can be accessed by the IBM Cloud Operator. Next, the script sets default values that are used to provision IBM Cloud services, like the resource group and region to provision the services in. Later, you can override any default value in the `Service` custom resource. Finally, the script deploys the operator in your cluster in the `ibmcloud-operators` namespace.
```

Comment:
I like the new wording 😄 
One note: the namespace name has changed to `ibmcloud-operator-system` and in OpenShift it's `openshift-operators`